scripts/git/common.ps1

```diff
@@ -55,9 +55,9 @@ function Get-StagedFiles {
 
 # Formats the given file using ClangFormat
 function ClangFormatFile([string]$repoRoot, [string]$filename, [bool]$isDryRun){
-    $clangCommand = "$repoRoot/tools/scripts/clang-format.ps1"
+    $clangCommand = "$repoRoot/scripts/git/clang-format.ps1"
```

Comment:
a.  We should update the script in a separate change. 
b.  This will merge to develop when packaging branch merges (or we cherry-pick this to develop if needed).  

b ==> a is critical. 


---
In reply to: [103279187](https://github.com/Microsoft/WinObjC/pull/2104#discussion_r103279187) [](ancestors = 103279187)